lib/splitio-api-binding.js

```diff
@@ -98,8 +98,45 @@ class SplitioApiBinding {
   /**
    * Get segment data.
    */
-  getSegments () {
-    throw new Error('Not implemented')
+  async getSegments ({ splits }) {
+    const allSegmentNames = []
+    const segments = {}
+    let usingSegmentsCount = 0
+    Object.values(splits).forEach(split => {
+      const segmentNames = getSegmentNamesInUse({
+        splitConditions: split.conditions
+      })
+      if (segmentNames.size) {
+        usingSegmentsCount++
+      }
+      segmentNames.forEach(segmentName => {
+        if (!allSegmentNames.includes(segmentName)) {
+          allSegmentNames.push(segmentName)
+        }
+      })
+    })
```

Comment:
Do you think making line 105-117 into `getSegmentNamesInUse()` will be better? I think if we don't really care the names within one split. Those process can be handled in that function. And we can just pass `splits` into it and `getSegmentNamesInUse()` will return `allSegmentNames` and `usingSegmentsCount`